README.md

```diff
@@ -68,43 +68,48 @@ Construct a client object with endpoint.
 
 ## Query
 
-`pymkts.Client#query(symbols, timeframe, attrgroup, start=None, end=None, limit=None, limit_from_start=False)`
+`pymkts.Client.query(symbols, timeframe, attrgroup, start=None, end=None, limit=None, limit_from_start=False)`
```

Comment:
I changed all of these to dots, feels more "Pythonic" to me, but I'm not especially attached to it either.